lucene/misc/src/java/org/apache/lucene/misc/store/DirectIODirectory.java

```diff
@@ -72,14 +64,17 @@
  * and OS X; other Unixes should work but have not been
  * tested!  Use at your own risk.
  *
+ * <p>Throws UnsupportedOperationException if the operating system or
```

Comment:
Can we move this to `@throws` javadoc tag?

Reply:
Ops. Fixed.